src/VirtualClient/VirtualClient.Actions/StressNg/StressNgExecutor.cs

```diff
@@ -49,13 +49,13 @@ public StressNgExecutor(IServiceCollection dependencies, IDictionary<string, ICo
         }
 
         /// <summary>
-        /// The name of the package where the SPECJbb package is downloaded.
+        /// The command line argument defined in the profile.
         /// </summary>
-        public int DurationInSecond
+        public string CommandLine
         {
             get
             {
-                 return this.Parameters.GetValue<int>(nameof(StressNgExecutor.DurationInSecond), DefaultRuntimeInSeconds);
+                return this.Parameters.GetValue<string>(nameof(StressNgExecutor.CommandLine));
             }
         }
 
```

Comment:
Update the summary for the IntializeAsync() method below.

Reply:
Corrected the summary